### PR TITLE
feat: update lightweight button and hyperlink underline size

### PR DESCRIPTION
**Pull request by bheston**
# Description

Updated lightweight button underline to 1px on hover/active, 2px on focus.
Updated hyperlink underline to 1px on rest/hover/active, 2px on focus.

## Motivation & context

The hover and active underline should match with normal outline widths (1px), only focus should use the focus width (2px).

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by chrisdholt** [comment]
Nit: Can you update the PR title to accurately reflect the component/appearance name (lightweight)?

File: packages/fast-components-styles-msft/src/button/index.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ const applyTransparentBackplateStyles: CSSRules<DesignSystem> = {
         boxShadow: "none",
         "& $button_contentRegion::before": {
             background: neutralForegroundRest,
+            height: toPx<DesignSystem>(focusOutlineWidth),
```
Comment (chrisdholt) [inline packages/fast-components-styles-msft/src/button/index.ts]: With this change, what is the difference between a hyperlink and our button component? Doesn't button just address hyperlink scenarios now?

Comment (bheston) [reply packages/fast-components-styles-msft/src/button/index.ts]: They were always visually consistent. As I understood, Hyperlink we more for use inline as an actual href and would take font styling from its parent, and Button is more for an actual semantic button that looks like a link. That is, not inline with other content.

Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/button/index.ts]: Button supports either an anchor or semantic button - these can look the same and function differently. The delta was that there was no underline on the button by default as there is no accessibility requirement for it as it is not inline with other text (as hypertext is). So, there was a visual delta between the two intentionally; the underline by default was on hypertext only as it is intended to be used inline with text.

Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/button/index.ts]: **Button:**
Screen Shot 2019-06-21 at 10 43 07 AM

**Hypertext**
Screen Shot 2019-06-21 at 10 44 50 AM

Comment (bheston) [reply packages/fast-components-styles-msft/src/button/index.ts]: Got it. It looks like my comment is misleading. There is still the same underline setup for the two components in default state. The material change is that _when_ there is an underline it is 1px _unless_ it's focus state, at which it's 2px.

Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/button/index.ts]: Gotcha - I'll pull down just to make sure I'm seeing things correctly and sign off :thumbsup:

Comment (bheston) [reply packages/fast-components-styles-msft/src/button/index.ts]: Title and comments updated.

**Pull request merged**